src/Compiler/FSComp.txt

```diff
@@ -1721,3 +1721,4 @@ featureAccessorFunctionShorthand,"underscore dot shorthand for accessor only fun
 3577,tcOverrideUsesMultipleArgumentsInsteadOfTuple,"This override takes a tuple instead of multiple arguments. Try to add an additional layer of parentheses at the method definition (e.g. 'member _.Foo((x, y))'), or remove parentheses at the abstract method declaration (e.g. 'abstract member Foo: 'a * 'b -> 'c')."
 featureUnmanagedConstraintCsharpInterop,"Interop between C#'s and F#'s unmanaged generic constraint (emit additional modreq)"
 3578,chkCopyUpdateSyntaxInAnonRecords,"This expression is an anonymous record, use {{|...|}} instead of {{...}}."
+3579,alwaysUseTypedStringInterpolation,"Prefer to use typed interpolated strings over untyped one."
```

Comment:
Not sure about this message. Maybe something like this would be a bit better?
```suggestion
3579,alwaysUseTypedStringInterpolation,"Consider using typed interpolated strings over untyped ones."
```

or

```suggestion
3579,alwaysUseTypedStringInterpolation,"Consider adding type identifiers to the interpolated string."
```